airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -193,7 +193,7 @@ protected List<TableInfo<CommonField<Datatype>>> discoverInternal(final JdbcData
                 .map(f -> {
                   final Datatype datatype = getFieldType(f);
                   final JsonSchemaType jsonType = getType(datatype);
-                  LOGGER.info("Table {} column {} (type {}[{}]) -> {}",
+                  LOGGER.info("Table {} column {} (type {}[{}], nullable {}) -> {}",
```

Comment:
Seeing as the `(type {}[{}], nullable {})` isn't _abundantly_ clear on what the format is can this also be updated to have explicit updates to say `column size`? I guess even the ending `-> {}` isn't explicitly clear that it represents `jsonType` information unless I knew this LOGGER line existed

Reply:
welp. I timed that merge very poorly. do you feel strongly enough to warrant opening a new pr >.>

fwiw the syntax `VARCHAR[123]` is probably not awful, since most SQL-based systems will use something like e.g. `NUMBER(42)` to represent a 42-digit number. `nullable false` is... not great, but my hope is that it's at least understandable?

Final log message should look something like:
```
Table THE_TABLE_NAME column THE_COLUMN_NAME (type VARCHAR[16777216], nullable false) -> {"type": "string"}
```
which is hopefully not awful

Reply:
Based on what you've showed with an example log message, would say the current change is sufficient